src/app/space/create/deployments/apps/deployment-card.component.spec.ts

```diff
@@ -50,6 +50,16 @@ class FakeDeploymentGraphLabelComponent {
   @Input() valueUpperBound: any;
 }
 
+@Component({
+  selector: 'deployment-status-icon',
+  template: ''
+})
+class FakeDeploymentStatusIconComponent {
+  @Input() cpuDataStream: Observable<CpuStat>;
+  icon: String;
+  toolTip: String;
```

Comment:
I think you can get away with only the @Inputs here - the other properties aren't visible to the parent component so nothing should break if they aren't there.